users/anderson/dmc12.h

```diff
@@ -0,0 +1,11 @@
+// Sexy LED animation.
+
+#include "quantum.h"
+#include "config.h"
```

Comment:
This is not needed, as `config.h` is included in all files by default, as part of the make system.

```suggestion
```